script/Arcueid/Day 11/11_00_ARC11_1.txt

```diff
@@ -408,17 +433,18 @@ The laid-back math teacher said absent-mindedly.
 [sha:c44476e28166ecc9462ab97e39811a5a275a0fc6]{
 -- Page 27, Offset 29555.
 -- 「やっぱビッチ系だったかあの肉食！」
+// Tempted to replace carnivore with cougar but she's only 27
```

Comment:
I think I leave it to your discretion. From the perspective of a high schooler 27 might as well be 45 so either way is fine